src/app.ts

```diff
@@ -54,7 +54,7 @@ const { initTokenCache } = frodo.cache;
     await initConnectionProfiles();
     await initTokenCache();
 
-    program.addCommand(admin());
+    admin(program);
```

Comment:
Not adding a child command, simply passing this command to a module to add its commands to it.